generators/kubernetes/templates/registry/_jhipster-registry.yml

```diff
@@ -84,17 +84,13 @@ spec:
         # Registry configuration
         - name: SPRING_PROFILES_ACTIVE
           value: prod,native
-        - name: SECURITY_USER_PASSWORD
+        - name: JHIPSTER_REGISTRY_PASSWORD
           valueFrom:
             secretKeyRef:
               name: registry-secret
               key: registry-admin-password
         - name: JHIPSTER_SECURITY_AUTHENTICATION_JWT_SECRET
           value: my-secret-token-to-change-in-production
-        - name: EUREKA_CLIENT_FETCH_REGISTRY
-          value: 'true'
-        - name: EUREKA_CLIENT_REGISTER_WITH_EUREKA
-          value: 'true'
```

Comment:
when resetting that, the UNKNOWN label turns to UP, but the registry is listed as a service.

the clustering itself work in both cases....how do we proceed here, then? @jdubois @PierreBesson